packages/rum/src/domain/record/observers.ts

```diff
@@ -203,22 +203,20 @@ function initMouseInteractionObserver(
     if (getNodePrivacyLevel(target, defaultPrivacyLevel) === NodePrivacyLevel.HIDDEN || !hasSerializedNode(target)) {
       return
     }
-    const { clientX, clientY } = isTouchEvent(event) ? event.changedTouches[0] : event
-    const position: MouseInteraction = {
-      id: getSerializedNodeId(target),
-      type: eventTypeToMouseInteraction[event.type as keyof typeof eventTypeToMouseInteraction],
-      x: clientX,
-      y: clientY,
-    }
-    if (window.visualViewport) {
-      const { visualViewportX, visualViewportY } = convertMouseEventToLayoutCoordinates(clientX, clientY)
-      position.x = visualViewportX
-      position.y = visualViewportY
+    const id = getSerializedNodeId(target)
+    const type = eventTypeToMouseInteraction[event.type as keyof typeof eventTypeToMouseInteraction]
+
+    let interaction: MouseInteraction
+    if (type !== MouseInteractionType.Blur && type !== MouseInteractionType.Focus) {
+      const { x, y } = computeCoordinates(event)
+      interaction = { id, type, x, y }
+    } else {
+      interaction = { id, type }
     }
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏we know that MouseEvent/TouchEvents might not have coordinates. While we are working on this, what about checking whether `clientX` / `clientY` are not undefined before proceeding to compute coordinates? I know @ThibautGeriz would be very happy about that :) 

A test could look like this:

```ts

 it('should ignore clicks events without coordinates', () => {
    a.dispatchEvent(new Event('click', { bubbles: true }))
    expect(mouseInteractionCallbackSpy).not.toHaveBeenCalled()
    expect(coordinatesComputed).toBeFalse()
  })
```

Reply:
I was planning to do a separate PR about this if you do not want to add more scope to this one as you want Bastien 🤷

Reply:
I'd be in favour of not handling it in this PR because:

- it would delay the resolution of this issue that prevents the customer raising the issue to use session replay
- it would make sense for the replay team to own this fix and monitor its impact on their telemetry

Reply:
Fine, let's keep things like that for now.